src/PKSim.Core/Snapshots/AlteredBuildingBlock.cs

```diff
@@ -0,0 +1,10 @@
+﻿using OSPSuite.Core.Domain;
+
+namespace PKSim.Core.Snapshots
+{
+   public class AlteredBuildingBlock
```

Comment:
I am saving a list of building block type and name that were altered in the simulation.